tower/src/util/boxed/sync.rs

```diff
@@ -18,7 +18,7 @@ use std::{
 /// If you need a boxed [`Service`] that implements [`Clone`] consider using
 /// [`BoxCloneService`](crate::util::BoxCloneService).
 ///
-/// See module level documentation for more details.
+/// See [module level documentation](super) for more details.
```

Comment:
`BoxService` is available as `tower::util::BoxService` as well as `tower::util::boxed::BoxService`, but the referenced documentation is at `tower::util::boxed`. Linking like this goes to `tower::util::boxed` regardless of where the user found `BoxService`.